keras_hub/src/utils/timm/convert_efficientnet.py

```diff
@@ -91,6 +91,53 @@
         "stackwise_nores_option": [True] + [False] * 5,
         "activation": "relu",
     },
+    "rw_m": {
+        "width_coefficient": 1.2,
+        "stackwise_depth_coefficient": [1.2] * 4 + [1.6] * 2,
+        "stackwise_kernel_sizes": [3, 3, 3, 3, 3, 3],
+        "stackwise_num_repeats": [2, 4, 4, 6, 9, 15],
+        "stackwise_input_filters": [24, 24, 48, 64, 128, 160],
+        "stackwise_output_filters": [24, 48, 64, 128, 160, 272],
+        "stackwise_expansion_ratios": [1, 4, 4, 4, 6, 6],
+        "stackwise_strides": [1, 2, 2, 2, 1, 2],
+        "stackwise_squeeze_and_excite_ratios": [0, 0, 0, 0.25, 0.25, 0.25],
+        "stackwise_block_types": ["fused"] * 3 + ["unfused"] * 3,
+        "stackwise_force_input_filters": [0, 0, 0, 0, 0, 0],
+        "stackwise_nores_option": [False] * 6,
+        "activation": "silu",
+        "num_features": 1792,
+    },
+    "rw_s": {
+        "width_coefficient": 1.0,
```

Comment:
I'd pass these the new way instead of the legacy and now undocumented path. `"stackwise...": [1.0] * 6`

Reply:
done